pkg/apis/eksctl.io/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -55,10 +55,10 @@ func (in *ClusterConfig) DeepCopyInto(out *ClusterConfig) {
 		*out = make([]string, len(*in))
 		copy(*out, *in)
 	}
-	if in.CertificateAuthorityData != nil {
-		in, out := &in.CertificateAuthorityData, &out.CertificateAuthorityData
-		*out = make([]byte, len(*in))
-		copy(*out, *in)
+	if in.Status != nil {
+		in, out := &in.Status, &out.Status
+		*out = new(ClusterStatus)
+		(*in).DeepCopyInto(*out)
```

Comment:
isn't this block the same as
```
out.Status = new(ClusterStatus)
in.Status.DeepCopyInto(out.Status)
```

Reply:
Maybe, but this code is generated by [`deepcopy-gen`](https://github.com/kubernetes/code-generator/tree/master/cmd/deepcopy-gen).

Reply:
:+1: I was wondering where it was _autogenerated_ from, maybe put that in the comment if you happen to come across next time

Reply:
Generaotor does leave a comment here: https://github.com/weaveworks/eksctl/blob/aca8ec28fa192b2f0653fb1e25ca3a756d8f6ef2/pkg/apis/eksctl.io/v1alpha3/zz_generated.deepcopy.go#L19

But there is a lot of noise, of course... And less obvious from the diff view. Not sure if we need to do anything about this right now, like automating code generation in CI.